apiary.apib

```diff
@@ -2474,6 +2474,22 @@ Create a new project.
         + title: `New company website` (string, required)
         + description (string, optional)
         + starts_on: `2016-02-04` (string, required)
+        + billing_method (object, optional)
+            + One Of
+                + Properties
+                    + method: `fixed` (string, required)
+                    + amount (Money, required)
+                + Properties
+                    + method: `time_material` (string, required)
```

Comment:
Does this mean `time or material`? And wouldn't it make more sense then to also name the property `time_or_material`? This looks a bit confusing to me now.

Reply:
It should actually be `time_and_materials` I think: https://en.wikipedia.org/wiki/Time_and_materials

Reply:
Free pro tip from the trenches: in absence of a real Domain Expert, Wikipedia is often your best friend.